### ISSUE TITLE
vagrant 1.8.1 sles 12 sp 1 configuring network dosen't seem to work

### ISSUE THREAD

**Issue by idar** [mitchellh/vagrant#7413]
### Vagrant version
vagrant 1.8.1

### Host operating system
Ubuntu 16.04

### Guest operating system
Sles 12 sp 1

Fails to bring up network interface for host only networking.
This has problably something to do with the new network setup thing on sp1 called wicked.

### Debug output
```
==> esa: Configuring and enabling network interfaces...
DEBUG ssh: Checking whether SSH is ready...
DEBUG ssh: Re-using SSH connection.
 INFO ssh: SSH is ready!
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute:  (sudo=false)
DEBUG ssh: Exit status: 0
DEBUG guest: Searching for cap: configure_networks
DEBUG guest: Checking in: suse
DEBUG guest: Found cap: configure_networks in suse
 INFO guest: Execute capability: configure_networks [#<Vagrant::Machine: esa (VagrantPlugins::ProviderVirtualBox::Provider)>, [{:type=>:static, :adapter_ip=>"192.168.33.1", :ip=>"192.168.33.11", :netmask=>"255.255.255.0", :auto_config=>true, :interface=>1}]] (suse)
DEBUG ssh: Checking whether SSH is ready...
DEBUG ssh: Re-using SSH connection.
 INFO ssh: SSH is ready!
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute:  (sudo=false)
DEBUG ssh: Exit status: 0
DEBUG guest: Searching for cap: network_scripts_dir
DEBUG guest: Checking in: suse
DEBUG guest: Found cap: network_scripts_dir in suse
 INFO guest: Execute capability: network_scripts_dir [#<Vagrant::Machine: esa (VagrantPlugins::ProviderVirtualBox::Provider)>] (suse)
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: touch /etc/sysconfig/network/ifcfg-eth1 (sudo=true)
DEBUG ssh: Exit status: 0
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: sed -e '/^#VAGRANT-BEGIN/,/^#VAGRANT-END/ d' /etc/sysconfig/network/ifcfg-eth1 > /tmp/vagrant-ifcfg-eth1 (sudo=true)
DEBUG ssh: Exit status: 0
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: cat /tmp/vagrant-ifcfg-eth1 > /etc/sysconfig/network/ifcfg-eth1 (sudo=true)
DEBUG ssh: Exit status: 0
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: rm -f /tmp/vagrant-ifcfg-eth1 (sudo=true)
DEBUG ssh: Exit status: 0
DEBUG ssh: Uploading: /tmp/vagrant20160610-1653-lvnb9t to /tmp/vagrant-network-entry_1
DEBUG ssh: Re-using SSH connection.
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: /sbin/ifdown eth1 2> /dev/null (sudo=true)
DEBUG ssh: Exit status: 0
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: cat /tmp/vagrant-network-entry_1 >> /etc/sysconfig/network/ifcfg-eth1 (sudo=true)
DEBUG ssh: Exit status: 0
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: /sbin/ifup eth1 2> /dev/null (sudo=true)
DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: stdout: eth1            no-device

DEBUG ssh: Exit status: 157
 INFO retryable: Retryable exception raised: #<Vagrant::Errors::VagrantError: The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

/sbin/ifup eth1 2> /dev/null

Stdout from the command:

eth1            no-device


Stderr from the command:
```

**Comment by sethvargo** [comment]
Hi @idar 

This will be fixed in Vagrant 1.8.2 (release later today or next week). We redid all the networking in guests.

**Comment by idar** [comment]
Thanks :+1:
